readthedocs/search/api/v3/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from readthedocs.search.api.v3.views import SearchAPI
+
+urlpatterns = [
+    path("", SearchAPI.as_view(), name="search_api_v3"),
```

Comment:
This feels like an odd pattern. Should we just include it directly?

Reply:
This is the same we are doing with the embed api https://github.com/readthedocs/readthedocs.org/blob/aa31543402a2c03e1f8a779e719266dc691b9671/readthedocs/embed/urls.py, and this also avoids having to repeat the `name="search_api_v3"` part when re-exporting it on .com.